app/src/main/java/com/simplecity/amp_library/ShuttleApplication.java

```diff
@@ -79,6 +81,16 @@
 
     private RefWatcher refWatcher;
 
+    private static ShuttleApplication instance;
+
+    public static ViewModelAdapter adapter = new ViewModelAdapter();
```

Comment:
Seems like these two variables should be inside synchronized methods too if they are static and shared across other classes?